cli/compose/convert/service.go

```diff
@@ -24,6 +24,7 @@ func Services(
 	namespace Namespace,
 	config *composetypes.Config,
 	client client.SecretAPIClient,
+	useServiceSpecNetworks bool,
```

Comment:
@aaronlehmann I would rather avoid the boolean arg and just pass the `ClientVersion` here, and handle the condition inside the func.